ingestion/src/metadata/ingestion/source/database/snowflake/queries.py

```diff
@@ -46,7 +46,7 @@
 )
 
 SNOWFLAKE_GET_TABLE_NAMES = """
-select TABLE_NAME from information_schema.tables where TABLE_SCHEMA = '{}' and TABLE_TYPE = 'BASE TABLE'
+select TABLE_NAME, IS_TRANSIENT from information_schema.tables where TABLE_SCHEMA = '{}' and TABLE_TYPE = 'BASE TABLE'
```

Comment:
maybe to make this a bit easier to read and more performant: Instead of getting the `IS_TRANSIENT` here and filtering after we get all the query results, could we have a couple of queries, one of them already filtering by `IS_TRANSIENT`?